src/pages/docs/chat/api/javascript/messages.mdx

```diff
@@ -0,0 +1,371 @@
+---
+title: Messages
+meta_description: ""
+---
+
+The `Messages` interface is used to send and subscribe to messages in a chat room.
+
+Access it via `room.messages`.
+
+---
+
+## Properties
+
+| Property | Description | Type |
+| -------- | ----------- | ---- |
+| reactions | Add, delete, and subscribe to message reactions. | [`MessageReactions`](/docs/chat/api/javascript/messages-reactions) |
+
+---
+
+## Send a message <a id="send"/>
+
+`room.messages.send()`
+
+Send a message in the chat room.
+
+This method uses the Ably Chat API endpoint for sending messages.
+
+Note that the Promise may resolve before OR after the message is received from the realtime channel. This means you may see the message that was just sent in a callback to `subscribe()` before the returned promise resolves.
+
+`send(params: SendMessageParams): Promise<Message>`
+
+### Parameters
+
+Use the following parameters:
+
+| Parameter | Description | Type |
+| --------- | ----------- | ---- |
+| `params` | An object containing the text, headers and metadata for the message. | [`SendMessageParams`](#sendmessageparams) |
+
+#### SendMessageParams
+
+The following `SendMessageParams` can be set:
+
+| Parameters | Description | Type |
+| ---------- | ----------- | ---- |
+| `text` | The text content of the message. | String |
+| `metadata` | *Optional* Metadata to attach to the message. Allows for attaching extra info to a message, which can be used for various features such as animations, effects, or simply to link it to other resources such as images, relative points in time, etc. This value is always set on received messages. If there is no metadata, this is an empty object. Do not use metadata for authoritative information. There is no server-side validation. When reading the metadata, treat it like user input. | [`MessageMetadata`](#metadata) |
+| `headers` | *Optional* Headers to attach to the message. Headers enable attaching extra info to a message, which can be used for various features such as linking to a relative point in time of a livestream video or flagging this message as important or pinned. This value is always set on received messages. If there are no headers, this is an empty object. Do not use the headers for authoritative information. There is no server-side validation. When reading the headers, treat them like user input. | [`MessageHeaders`](#headers) |
+
+#### MessageMetadata | Metadata <a id="metadata"/>
```

Comment:
I think we should drop the `|` section, the fact that `MessageMetadata` is currently an alias shouldn't be part of the API docs as we could change it